From 2ce371301192cf558ba184ef7177d5654ebb6847 Mon Sep 17 00:00:00 2001
From: Balazs Meszaros <balazs.meszaros@cloudera.com>
Date: Tue, 23 May 2017 20:41:14 +0200
Subject: [PATCH 352/362] HBASE-18096 Limit HFileUtil visibility and add
 missing annotations

Signed-off-by: Chia-Ping Tsai <chia7712@gmail.com>

(cherry picked from commit cb136d8d25568a97b0f8568178a7eb48b42bba89)

Change-Id: I4a0dde361eb553f2accc45d15ed14c73c24f6330
Author: Balazs Meszaros
Reason: Bug
Ref: CDH-54278
---
 .../apache/hadoop/hbase/io/hfile/HFileUtil.java    |    4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileUtil.java b/hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileUtil.java
index 835450c..83feb72 100644
--- a/hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileUtil.java
+++ b/hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFileUtil.java
@@ -20,8 +20,10 @@ package org.apache.hadoop.hbase.io.hfile;
 import java.io.IOException;
 
 import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.hbase.classification.InterfaceAudience;
 
-public class HFileUtil {
+@InterfaceAudience.Private
+class HFileUtil {
 
   /** guards against NullPointer
    * utility which tries to seek on the DFSIS and will try an alternative source
-- 
1.7.9.5

